### PR TITLE
reorganizing collecting information java

### PR DESCRIPTION
**Pull request by daschl**
ping @simonbasle @ingenthr 

I put the logging stuff on top and updated the samples on latency metrics since the output has changed.

### CODE REVIEW

File: content/sdk/java/collecting-information.dita [hunk 1]
```diff
@@ -3,31 +3,160 @@
   PUBLIC "-//OASIS//DTD DITA Concept//EN" "concept.dtd">
 <concept xml:lang="en-us" id="concept2677">
 	<title>Collecting Information</title>
-		
-	<shortdesc>The Java SDK provides an event bus that transmits information about the behavior of your
+
+	<shortdesc>The Java SDK logs events and also provides an event bus that transmits information about the behavior of your
 		database system, including system and metric events. It has no hard dependency on a specific logger implementation.</shortdesc>
 	<conbody>
```
Comment (simonbasle) [inline content/sdk/java/collecting-information.dita]: I would end the previous sentence with something like "but it is recommended you add one (the SDK will adapt to standard implementations like Log4J and SLF4J)"...

File: content/sdk/java/collecting-information.dita [hunk 2]
```diff
@@ -3,31 +3,160 @@
...
+      <title>Manual Overriding</title>
+
+      <p>If different loggers are on the class path and you want to force a different one (for example
+        Log4J instead of SLF4J) you can set the default one directly:</p>
+
+      <codeblock outputclass="language-java"><![CDATA[CouchbaseLoggerFactory.setDefaultFactory(new Log4JLoggerFactory());]]></codeblock>
+
+    </section>
+
+
```
Comment (simonbasle) [inline content/sdk/java/collecting-information.dita]: maybe add a section "loggers of interest" or similar. For example, these two logger configurations can be useful (taken from the test log4j configuration):

```ini
# the following forces logging of BucketManager's N1QL index watcher steps
log4j.logger.indexWatch=DEBUG
# the following avoid seeing Node connect/disconnect logs
log4j.logger.com.couchbase.client.core.node.Node=OFF
```

File: content/sdk/java/collecting-information.dita [hunk 3]
```diff
@@ -3,31 +3,160 @@
...
+  <p><b>Configuring SLF4J</b></p>
+
+
+      <p>To enable SLF4J, put it on the class path, as well as one of the support logger
+        implementations (like logback). If you want to use logback and include logback-classic,
+        it will be pulled in automatically:</p>
+
+      <codeblock outputclass="language-xml"><![CDATA[<dependency>
```
Comment (simonbasle) [inline content/sdk/java/collecting-information.dita]: aligning of rows 2+ are all a bit off in all these codeblocks

Comment (ingenthr) [inline content/sdk/java/collecting-information.dita]: It was that the first line needed two spaces.

File: content/sdk/java/collecting-information.dita (hunk 2 above)
Comment (ingenthr) [inline content/sdk/java/collecting-information.dita]: Added those there, but this should be updated a bit further at some point.

**Comment by ingenthr** [comment]
Considered Simon's inputs, rebased, reviewed, merged.

**Pull request closed without merging**